configurer/path_darwin.go

```
@@ -3,10 +3,19 @@
 package configurer
 
 import (
+	"fmt"
 	"os"
 	"path/filepath"
+
+	"github.com/pastelnetwork/pastel-utility/constants"
 )
 
+// GetHomeDir returns the home path for darwin OS.
+func GetHomeDir() string {
+	homeDir, _ := os.UserConfigDir()
```

Please never ignore errors in Go